doc/events.md

```diff
@@ -45,8 +44,8 @@ that anonymous inner classes cannot be used as event subscribers.
 Here the `DocumentNotificationClient` is subscribing to the `DocumentCreated` event. Because the
 events are just classes, they obey the type hierarchy. This means that it is also possible to
 subscribe to the generic `DocumentModified` event to be notified of all Created/Deleted/Updated
-events and then use reflection and downcasting if needed, however if you want to listen for multiple 
-event types eg `DocumentCreated` and `DatastoreCreated` using the same subscriber method, the 
+events and then use reflection and downcasting if needed. However if you want to listen for multiple 
+event types eg `DocumentCreated` and `DocumentStoreCreated` using the same subscriber method, the 
```

Comment:
We've had discussions about this before and in the absence of a style guide I prefer it without punctuation. We don't write "I.B.M." or "to-day" in modern writing.

[The Guardian](https://www.theguardian.com/guardian-observer-style-guide-e) says "no full points", but it appears that [The Chicago Manual of Style](http://www.chicagomanualofstyle.org/qanda/data/faq/topics/Abbreviations/faq0047.html) does use them.